samples/StandaloneApp/descriptors/StandaloneApp.xml

```diff
@@ -0,0 +1,3 @@
+<linker>
+	<assembly fullname="StandaloneApp" />
```

Comment:
I'm curious about why this is necessary and doesn't get generated during the build like it does for other projects.